### DISCUSSION TITLE
What does brew mean by, "New Formulae" and "New Casks?"

### DISCUSSION THREAD

**Discussion by simurgh9** [Homebrew/discussions#4859] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.17
ORIGIN: https://github.com/Homebrew/brew
HEAD: 35746e0a6ba6c3c5cfe56d99f79d9ec9f52ee15f
Last commit: 2 days ago
Core tap JSON: 24 Oct 20:42 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: emacsclient -a emacs
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.0-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

This is not as much of an issue as it is a question. When I run, `brew update`, I sometimes see "New Formulae" and "New Casks" section listed along with "Outdated Formulae" etc. Bellow is an example,

<img width="739" alt="sc 1" src="https://github.com/Homebrew/discussions/assets/43047754/42190dcb-9274-4cb9-a47b-d8e58645fbc8">

What are the "New Formulae" and "New Casks" sections listing? Are these the new dependencies of the packages to be updated? Or, are these just advertisements for new software added to Brew?

If these are just advertisements, is there anyway to turn these off?

**Comment by ZhongRuoyu** [comment]
They are the new packages added since you last updated. To turn these off, set the `HOMEBREW_NO_UPDATE_REPORT_NEW` environment variable to a non-empty value (e.g., `1`).

From https://docs.brew.sh/Manpage#environment:

> `HOMEBREW_NO_UPDATE_REPORT_NEW`
> If set, brew update will not show the list of newly added formulae/casks.

**Comment by simurgh9** [comment]
Okay thank you :)

**Comment by lafkpages** [comment]
Is there a way to list these manually?

**Reply by gabow93** [reply]
im also very much in search of this possibility

**Reply by SMillerDev** [reply]
No, because it's dependent on the changes that happened between updates.
